apps/desktop/src/lib/components/ProjectSetup.svelte

```diff
@@ -27,7 +27,7 @@
 		loading = true;
 		try {
 			// TODO: Refactor temporary solution to forcing Windows to use system executable
-			if ($platformName === 'win32') {
+			if (platformName === 'windows') {
```

Comment:
have we tested that this works?

Reply:
They are enumerated so typescript complains if you are using an incorrect value.